### PR TITLE
Main form names

### PR DESCRIPTION
**Pull request by lilyclements**

### CODE REVIEW

**Comment by dannyparsons** [comment]
I think @FrancoisJRenaud is editing frmMain on one of his pull requests so I will leave this open until that is merged then check if this can be.

**Comment by lilyclements** [comment]
Great. These are all small things that can easily be re-done!

File: instat/frmMain.Designer.vb [hunk 1]
```diff
@@ -2727,40 +2727,40 @@ Partial Class frmMain
...
         Me.mnuOrganiseColumnGenerateRandomSamples.Text = "Random Samples..."
         '
         'mnuOrganiseColumnGeneratePermuteRows
         '
         Me.mnuOrganiseColumnGeneratePermuteRows.Name = "mnuOrganiseColumnGeneratePermuteRows"
-        Me.mnuOrganiseColumnGeneratePermuteRows.Size = New System.Drawing.Size(177, 22)
+        Me.mnuOrganiseColumnGeneratePermuteRows.Size = New System.Drawing.Size(179, 22)
         Me.mnuOrganiseColumnGeneratePermuteRows.Tag = "Permute_Rows..."
-        Me.mnuOrganiseColumnGeneratePermuteRows.Text = "Permute Rows..."
+        Me.mnuOrganiseColumnGeneratePermuteRows.Text = "Permute Columns..."
```
Comment (dannyparsons) [inline instat/frmMain.Designer.vb]: I think it's probably "Permute Column" but could check with @rdstern.

Comment (rdstern) [inline instat/frmMain.Designer.vb]: Yes, I think it is a single receiver.

**Pull request closed without merging**